Box.V2.Test/BoxUsersManagerTest.cs

```diff
@@ -51,7 +51,7 @@ public async Task GetUserInformation_WithField_ValidResponse_ValidUser()
         {
             /*** Arrange ***/
             IBoxRequest boxRequest = null;
-            string responseString = "{\"type\": \"user\", \"id\": \"12345\", \"status\": \"active\"}";
+            string responseString = "{\"type\": \"user\", \"id\": \"12345\", \"status\": \"active\", \"notification_field\": []}";
```

Comment:
Also, can we parameterize the responseString for this method so we can call it twice: once with this "empty array" string and once with "null" for notification_email?

Reply:
I test the null case in a separate test method. It would make sense to parameterize if this was long term, but we are probably going to remove this in a quarter or two when the API is fixed.